f5/bigip/tm/security/firewall.py

```diff
@@ -232,3 +233,11 @@ def __init__(self, global_rules):
         super(Global_Rules, self).__init__(global_rules)
         self._meta_data['required_json_kind'] = \
             'tm:security:firewall:global-rules:global-rulesstate'
+
+
+class Global_Fqdn_Policy(UnnamedResource):
+    """BIG-IP® AFM® Global FQDN policy resource"""
+    def __init__(self, global_fqdn_policy):
```

Comment:
if this collection works really on 12.x then minimal version needs to be added here, by default we have it set on 11.5.0:

https://github.com/F5Networks/f5-common-python/blob/development/f5/bigip/resource.py#L169

You set it on collections (not Organizing Collections) or unnamed resources:

https://github.com/F5Networks/f5-common-python/blob/development/f5/bigip/tm/asm/policies/vulnerabilities.py#L27

Reply:
Added in version check for Global FQDN policy